### PR TITLE
Add content to README

### PR DESCRIPTION
**Pull request by kirstenmg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Animated Bar Chart Racer
+
+## Description
+
+This animated bar chart racer visualizes how data change over time. It displays bar charts for each point in time and maintains them in sorted order, so as the value of one bar surpasses another, the bars switch and they appear to be “racing” (see the example below).
```
Comment (linasc3-ai) [inline README.md:5]: typo - should be how data _changes_

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Animated Bar Chart Racer
+
+## Description
+
+This animated bar chart racer visualizes how data change over time. It displays bar charts for each point in time and maintains them in sorted order, so as the value of one bar surpasses another, the bars switch and they appear to be “racing” (see the example below). 
+
+Our implementation also provides an interface for the user to upload a formatted csv file, choose values for parameters such as axis labels or frame count, and view and export the final animation.
```
Comment (linasc3-ai) [inline README.md:7]: should we make it more convincing + specific? we could use bullet point format and write something like "Our implementation provides an interactive interface that allows the user to: 
-  upload a csv file and properly format it using x
-  choose values for parameters such as chart titles, axis labels, and frame count (to customize the speed of animation)
- choose a color to customize the chart theme 
- view and export the final animation as a .gif"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+You can find more examples of saved animations in the [examples](/examples) folder.
+
+### Demonstration video
+[![Demo Video](https://img.youtube.com/vi/Asjtc1Ev0e8/default.jpg)](https://youtu.be/Asjtc1Ev0e8)
+
+This video was created for CodeLabs Demo Day and provides a brief demonstration of using the program.
+
+## Authors and acknowledgement
+This program was created as part of CodeLabs, an internship through CodeDay. Our interns are Anusha Puri, Lina Chihoub, and Kirsten Graham. Our mentor is Charlie Liu.
```
Comment (CLiu13) [inline README.md:25]: Suggestion: hyperlink each of our names to our GitHub profile (or LinkedIn profile, up to you) - that way, if anyone looking through wants to learn more, they can easily find you 🙂

File: README.md (hunk 2 above)
Comment (kirstenmg) [reply README.md]: Good idea - it has been added!

File: README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+This animated bar chart racer visualizes how a dataset changes over time. It displays bar charts for each point in time and maintains them in sorted order, so as the value of one bar surpasses another, the bars switch and they appear to be “racing” (see the example below). 
+
+Our implementation provides an interface that allows the user to:
+ - import data by uploading a csv file
+ - choose values for parameters such as chart title, axis labels, and frame count (to customize the speed of animation)
+ - choose a color to customize the chart theme
+ - view and export the final animation as a gif or mp4 file
+
+This program was coded in Python, using tkinter for the interface, pandas for data processing, and matplotlib for graphing and animation.
```
Comment (CLiu13) [inline README.md:13]: comma after Python isn't necessary AFAIK, but that's just me being nit-picky with grammar :P

File: README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+You can find more examples of saved animations in the [examples](/examples) folder.
+
+### Demonstration video
+[![Demo Video](https://img.youtube.com/vi/Asjtc1Ev0e8/default.jpg)](https://youtu.be/Asjtc1Ev0e8)
+
+This video was created for CodeLabs Demo Day and provides a brief demonstration of using the program.
+
+## Authors and acknowledgement
+This program was created as part of CodeLabs, an internship through CodeDay. Our interns are [Anusha Puri](https://github.com/puria123), [Lina Chihoub](https://github.com/linasc3-ai), and [Kirsten Graham](https://github.com/kirstenmg). Our mentor is [Charlie Liu](https://github.com/CLiu13).
```
Comment (CLiu13) [inline README.md:29]: ```CodeLabs, an internship through CodeDay``` -> ```CodeLabs, a virtual internship program organized by CodeDay.```

Comment (CLiu13) [reply README.md:29]: Also, might be worth hyper-linking to the CodeLabs page so any future employers/viewers can learn more about the program if they are curious

**Pull request merged**